modules/Microsoft.Compute/sshPublicKeys/readme.md

```diff
@@ -0,0 +1,253 @@
+# Compute Disks `[Microsoft.Compute/sshPublicKeys]`
+
+This template deploys a disk
```

Comment:
```suggestion
This template deploys a Public SSH Key resource.
> Note: It does not generate the key for you but acts as a store for a provided key. The key can be generated using for example a PowerShell script in a Deployment Script.

```

Reply:
Does the invoke command output a private key? the use case here is to generate the key pair, the sshpublickey holds the public key property which is used when deploying the VM. The private key is stored locally in a file .\ssh\filename and we get that private key and store it in keyvault. it is used for sshlogon to Linux.